### PR TITLE
Reformat UG

### PR DESCRIPTION
**Pull request by rui-han-crh**
Changes instance of "address book" to "SectresBook". Please refer to changed files.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -169,14 +177,14 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+SectresBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+AddressBook data are saved as a JSON file `[JAR file location]/data/sectresbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (ryanczx) [inline docs/UserGuide.md:184]: Line 184, 
`AddressBook data are saved as a JSON file...`

Update `AddressBook` to `SectresBook`

**Review by ryanczx — CHANGES_REQUESTED** [review]
Update line 184.
Other than that, lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (rui-han-crh) [reply docs/UserGuide.md]: Updated

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,7 +129,7 @@ Examples:
 
 ### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified person from the aSectresBook.
```
Comment (ryanczx) [inline docs/UserGuide.md:132]: Sorry, just saw this.
Change `aSectresBook` to `SectresBook`

**Comment by ryanczx** [comment]
Line 89: `t\Human Resources` may cause a merge conflict?
Maybe change it to `t/HumanResources`

File: docs/UserGuide.md (hunk 2 above)
Comment (rui-han-crh) [reply docs/UserGuide.md]: Yikes I didnt see that either, updated and pushed

**Comment by rui-han-crh** [comment]
> Line 89: `t\Human Resources` may cause a merge conflict? Maybe change it to `t/HumanResources`

Hmm, I don't think this will conflict, if I don't touch the line, it should merge into the latest version which is yours with `t/HumanResources`. So currently there's no indication of merge conflict yet with the status below and yours in already in main, so I think it's safe

**Comment by ryanczx** [comment]
>Hmm, I don't think this will conflict, if I don't touch the line, it should merge into the latest version which is yours with `t/HumanResources`. So currently there's no indication of merge conflict yet with the status below and yours is already in main, so I think it's safe

Ok, then it should be fine!

**Review by ryanczx — APPROVED** [review]
lgtm

**Review by Pinran-J — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -28,7 +28,9 @@ SectresBook helps **secretaries** to maintain all the information of the members
 
    * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`delete 3`** : Deletes the 3rd contact shown in the current list.
+
```
Comment (czhongwei) [inline docs/UserGuide.md:32]: Can add the `find` command

Comment (rui-han-crh) [reply docs/UserGuide.md:32]: wait what does this mean?

**Comment by rui-han-crh** [comment]
Can someone please merge, I don't think there's any pending issues with the lines that have been changed in this PR thank u

**Pull request merged**